### ISSUE TITLE
hello haw i can solve this issu

### ISSUE THREAD

**Issue by foudilbenabid** [zeyad-mansour/lunar#288]
(Neural Network Aimbot)
[INFO] Loading the neural network model
Downloading: "https://github.com/ultralytics/yolov5/zipball/master" to C:\Users\Houd/.cache\torch\hub\master.zip
YOLOv5  2023-9-15 Python-3.11.5 torch-2.0.1+cpu CPU

Fusing layers...
Model summary: 224 layers, 7053910 parameters, 0 gradients
Adding AutoShape...
[!] CUDA ACCELERATION IS UNAVAILABLE
[!] Check your PyTorch installation, else performance will be poor

[INFO] PRESS 'F1' TO TOGGLE AIMBOT
[INFO] PRESS 'F2' TO QUIT
[INFO] Beginning screen capture
[!] AIMBOT IS [ENABLED]

**Comment by foudilbenabid** [comment]
(Neural Network Aimbot)
[INFO] Loading the neural network model
Downloading: "https://github.com/ultralytics/yolov5/zipball/master" to C:\Users\Houd/.cache\torch\hub\master.zip
YOLOv5  2023-9-15 Python-3.11.5 torch-2.0.1+cpu CPU

Fusing layers...
Model summary: 224 layers, 7053910 parameters, 0 gradients
Adding AutoShape...
[!] CUDA ACCELERATION IS UNAVAILABLE
[!] Check your PyTorch installation, else performance will be poor

[INFO] PRESS 'F1' TO TOGGLE AIMBOT
[INFO] PRESS 'F2' TO QUIT
[INFO] Beginning screen capture
[!] AIMBOT IS [ENABLED]
